tests/lib/linter/linter.js

```diff
@@ -17505,32 +17505,6 @@ var a = "test2";
         });
     });
 
-    describe("Mutability", () => {
```

Comment:
Not sure why these are still here. There's no way to test this using flat config.